scripts/pin_the_slow_way.py

```diff
@@ -76,8 +76,8 @@
           'ncurses': 'ncurses 5.9',  # 5.9
           'netcdf-cxx4': 'netcdf-cxx4 4.3.*',  # NA
           'netcdf-fortran': 'netcdf-fortran 4.4.*',  #
+          'openblas': 'openblas 0.2.20|0.2.20.*',  # 0.2.19
```

Comment:
@isuruf is this OK to merge?

Reply:
Yes, it is. Maybe we want to add a comment that the runtime requirement can be relaxed.

Reply:
I am on the fence about that. We don't support that "officially" b/c (a) the pinning bot is dead, (b) even when active it could not do that.

It is an advanced technique that a comment might encourage it, and I'd rather leave that to those who a certain about what they are doing.

Reply:
Okay. I thought it would be better, since we do the same thing for numpy.